src/components/Widget/components/Conversation/components/Header/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+const close = require('../../../../../../../assets/clear-button.svg') as string;
+
+import './style.scss';
+
+type Props = {
+  title: string;
+  subtitle: string;
+  toggleChat: () => void;
+  showCloseButton: boolean;
+  titleAvatar?: string;
+}
+
+function Header({ title, subtitle, toggleChat, showCloseButton, titleAvatar }: Props) {
+  return (
+    <div className="rcw-header">
+      {showCloseButton &&
+        <button className="rcw-close-button" onClick={toggleChat}>
+          <img src={close} className="rcw-close" alt="close" />
```

Comment:
This is not related to this pull request, but the alt text should be translated. Do you have a way of passing locales or texts?

Reply:
Maybe I can add a prop to manage that, but I think I'll need to add i18next and translate all alts to some extent 🤔 . Let me see if I can do it in another PR